NOpenCL/CommandQueue.cs

```diff
@@ -8,7 +8,10 @@ namespace NOpenCL
     using System;
     using System.ComponentModel;
     using NOpenCL.SafeHandles;
+    using System.Diagnostics;
+    using System.Runtime.InteropServices;
 
+    [DebuggerDisplay("Device = {Device.Name}")]
```

Comment:
@SunsetQuest It can go either way really. The advantage of sending single-topic (smaller) pull requests is it makes it easier to quickly merge items while other proposals are still under review. For example, based on your answer to my questions I would be ready to merge the debugger display improvements right now, but I can't because they were combined with other work that I haven't made decisions on yet.